x/staking/client/cli/cli_test.go

```diff
@@ -1260,6 +1263,75 @@ func (s *IntegrationTestSuite) TestNewCmdUnbond() {
 	}
 }
 
+// TestBlockResults tests that the validator updates correctly show when
+// calling the /block_results RPC endpoint.
+// ref: https://github.com/cosmos/cosmos-sdk/issues/7401.
+func (s *IntegrationTestSuite) TestBlockResults() {
+	val := s.network.Validators[0]
```

Comment:
Whenever I see / read `val` I think it's a value, not a validator. How about renaming to `v`?

Reply:
I personally slightly prefer `val`, but I also don't have a strong opinion, `v` is fine too.

However, we do use `val` everywhere, and for consistency, would prefer to put `val` here. Maybe a new PR to change `s/val/v/g` would make more sense?

Reply:
Yes, I've noticed that `val` is used often. So let's keep it for consistency.